codesign/lib/src/stdio.dart

```diff
@@ -0,0 +1,105 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io' as io;
+
+import 'package:meta/meta.dart';
+
+abstract class Stdio {
```

Comment:
Does the codesign tool need its own stdio? Can we use package:logging instead?

Reply:
umm Yeah currently we are overriding stdio, and we can use test Logger to override logger similar to https://github.com/flutter/cocoon/blob/dc6bde54efcb0310670823f3bd38006ad15b651b/device_doctor/test/src/utils.dart#L83-L120. shall we override stdio or logger?